polkadot/zombienet_tests/elastic_scaling/0002-elastic-scaling-doesnt-break-parachains.toml

```diff
@@ -36,5 +36,5 @@ onboard_as_parachain = false
 
   [parachains.collator]
   name = "collator2000"
-  command = "polkadot-parachain"
+  command = "polkadot-parachain-omni-node"
```

Comment:
I see `chain = "rococo-local"` in this file. We should slowly move all of these to something like `chain = "./rococo-local.json"`.

Reply:
IMO, we should keep the relay chains around. I mean there are not that many as parachains and it just makes it easier.

Reply:
Possibly behind a feature flag, the compile time is pretty shit 🙈

Reply:
Looks like a bigger change, or at least a verbose one. I think we should explore it ain a separate PR/issue